### PR TITLE
style(*) cleanup to match existing style guidelines

### PR DESCRIPTION
**Pull request by p0pr0ck5**
### Summary

Broad brush stroke to match documented style of avoiding one-line conditionals and enforcing spacing around string concatenation.

### CODE REVIEW

File: kong/conf_loader.lua [hunk 1]
```diff
@@ -346,7 +346,9 @@ local function load(path, custom_conf)
   local s = pl_stringio.open(kong_default_conf)
   local defaults, err = pl_config.read(s)
   s:close()
-  if not defaults then return nil, "could not load default conf: "..err end
+  if not defaults then
+    return nil, "could not load default conf: "..err
```
Comment (thibaultcha) [inline kong/conf_loader.lua]: Granted this is about 1 liners, but I think we are zealously trying to play by the book here. 

Adding spaces around the `..` operator is still part of the same scope related to "style improvements of the codebase conditions", and avoids 2 commits to change the same line unnecessarily.

File: kong/plugins/galileo/buffer.lua [hunk 2]
```diff
@@ -254,7 +256,9 @@ function _M:add_entry(...)
 
   if err >= self.queue_size then -- err is the queue size in this case
      ok, err = self:flush()
-     if not ok then return nil, err end -- for our tests only
+     if not ok then
+       return nil, err
+     end -- for our tests only
```
Comment (thibaultcha) [inline kong/plugins/galileo/buffer.lua]: We should probably move this comment inside the branch, as per tacit convention across the codebase.

File: kong/plugins/jwt/jwt_parser.lua [hunk 3]
```diff
@@ -191,7 +197,9 @@ _M.__index = _M
 -- @return JWT parser
 -- @return error if any
 function _M:new(token)
-  if type(token) ~= "string" then error("Token must be a string, got "..tostring(token), 2) end
+  if type(token) ~= "string" then
+    error("Token must be a string, got "..tostring(token), 2)
```
Comment (thibaultcha) [inline kong/plugins/jwt/jwt_parser.lua]: ditto: `..` operator

File: kong/plugins/ldap-auth/asn1.lua [hunk 4]
```diff
@@ -127,7 +127,9 @@ _M.ASN1Decoder = {
     while (sPos < len) do
       local newSeq
       sPos, newSeq = self:decode(sStr, sPos)
-      if (not(newSeq) and self.stoponerror) then break end
+      if (not(newSeq) and self.stoponerror) then
```
Comment (thibaultcha) [inline kong/plugins/ldap-auth/asn1.lua]: This file should be a vendor file, and not included in the plugins namespace :/ Too late now.

I forgot this rule in the contributing guide: no parenthesis in expressions, especially when they don't help clarifying said expression as is the case here.

File: kong/plugins/request-transformer/access.lua [hunk 5]
```diff
@@ -331,7 +335,9 @@ end
 local function transform_body(conf)
   local content_type_value = req_get_headers()[CONTENT_TYPE]
   local content_type = get_content_type(content_type_value)
-  if content_type == nil or #conf.rename.body < 1 and #conf.remove.body < 1 and #conf.replace.body < 1 and #conf.add.body < 1 and #conf.append.body < 1 then return end
+  if content_type == nil or #conf.rename.body < 1 and #conf.remove.body < 1 and #conf.replace.body < 1 and #conf.add.body < 1 and #conf.append.body < 1 then
```
Comment (thibaultcha) [inline kong/plugins/request-transformer/access.lua]: Again, a style update touching the conditions would - assuming we're not that zealous about commit atomicity - require that once touched, the conditions should match the required code style. We do get rid of the one-liner here but we should also get rid of the > 80 chars. Killing 2 birds at once like so would still be enclosed under the same scope of "improving the conditions code style".

File: kong/tools/utils.lua [hunk 6]
```diff
@@ -478,8 +496,12 @@ _M.normalize_ipv6 = function(address)
     port = nil
   end
   -- check ipv6 format and normalize
-  if check:sub(1,1) == ":" then check = "0"..check end
-  if check:sub(-1,-1) == ":" then check = check.."0" end
+  if check:sub(1,1) == ":" then
+    check = "0"..check
```
Comment (thibaultcha) [inline kong/tools/utils.lua]: ditto: concat

**Comment by p0pr0ck5** [comment]
Yeah, we can do a few more cleanup commits and make this an overall style cleanup PR.

**Comment by thibaultcha** [comment]
Not talking about a full cleanup PR necessarily, but rather a cleanup of the statements in a single commit.

**Comment by p0pr0ck5** [comment]
^ I think we get into some pretty ugly scope creep here. But I do get what you're saying. In addition to the two remaining open PR comments, let me know your thoughts on this PR as it stands.

File: kong/plugins/galileo/buffer.lua [hunk 7]
```diff
@@ -144,7 +144,7 @@ _send = function(premature, self, to_send)
         log(DEBUG, "Galileo collector saved the ALF (200 OK): ", body)
       elseif res.status == 207 then
         log(DEBUG, "Galileo collector partially saved the ALF "
-                 .."(207 Multi-Status): ", body)
+                 .. "(207 Multi-Status): ", body)
```
Comment (thibaultcha) [inline kong/plugins/galileo/buffer.lua]: *cries* Let's keep it like this for now, but damn...

File: kong/plugins/response-ratelimiting/schema.lua [hunk 8]
```diff
@@ -12,11 +12,11 @@ local function check_ordered_limits(limit_value)
     if limit_value[v] then
       has_value = true
       if limit_value[v] <=0 then
-        invalid_value = "Value for "..v.." must be greater than zero"
+        invalid_value = "Value for " .. v.. " must be greater than zero"
```
Comment (thibaultcha) [inline kong/plugins/response-ratelimiting/schema.lua]: missing one prior

File: kong/serf.lua [hunk 9]
```diff
@@ -16,7 +16,7 @@ Serf.__index = Serf
 Serf.args_mt = {
   __tostring = function(t)
     local buf = {}
-    for k, v in pairs(t) do buf[#buf+1] = k.." '"..v.."'" end
+    for k, v in pairs(t) do buf[#buf+1] = k .. " '" .. v.. "'" end
```
Comment (thibaultcha) [inline kong/serf.lua]: missing one prior

File: kong/tools/utils.lua [hunk 10]
```diff
@@ -603,12 +603,12 @@ _M.format_host = function(p1, p2)
     host = p1
     typ = _M.hostname_type(host)
   else
-    return nil, "cannot format type '"..t.."'"
+    return nil, "cannot format type '" .. t.. "'"
```
Comment (thibaultcha) [inline kong/tools/utils.lua]: idem

File: kong/tools/utils.lua [hunk 11]
```diff
@@ -603,12 +603,12 @@ _M.format_host = function(p1, p2)
...
   end
   if (typ == "ipv6") and (not find(host, "%[")) then
-    return "["..host.."]" ..  (port and ":"..port or "")
+    return "[" .. host .. "]" ..  (port and ":" .. port or "")
```
Comment (thibaultcha) [inline kong/tools/utils.lua]: can we also remove the double space one? since we're at it... 💤

File: kong/cmd/utils/nginx_signals.lua [hunk 12]
```diff
@@ -33,12 +33,12 @@ end
 
 local function send_signal(kong_conf, signal)
   if not kill.is_running(kong_conf.nginx_pid) then
-    return nil, "nginx not running in prefix: "..kong_conf.prefix
+    return nil, "nginx not running in prefix: " .. kong_conf.prefix
```
Comment (thibaultcha) [inline kong/cmd/utils/nginx_signals.lua]: in this file, we are missing L24 and L65

**Comment by thibaultcha** [comment]
There are a bunch missing in places not seen by this diff, like the conf_loader, some schemas, etc... We can see those with some grepping.

BTW: this must have been painful.

**Comment by p0pr0ck5** [comment]
A few of these were missed, needed to tweak the regex and re-apply.

Also added a new commit on this branch to remove wrapping parens around conditionals.

**Comment by thibaultcha** [comment]
I think we are still missing a few cases:
```
$ grep -n -R '"\.\.$' kong
kong/cmd/utils/prefix_handler.lua:65:      log.verbose("'resty' found at %s uses incompatible OpenResty. Kong "..
kong/cmd/utils/prefix_handler.lua:89:    return nil, ("could not find OpenResty 'resty' executable. Kong requires"..
kong/core/globalpatches.lua:213:          ngx.log(ngx.WARN, debug.traceback("math.randomseed() must be "..
kong/serf.lua:145:        log.warn("could not join cluster at %s, if the node does not exist "..
```

Do we want to take care of those?

**Comment by p0pr0ck5** [comment]
Grabbed these plus a few more

**Pull request merged**